x-pack/elastic-agent/spec/filebeat.yml

```diff
@@ -80,6 +80,7 @@ rules:
     - udp
     - unix
     - winlog
+    - filestream
```

Comment:
Do we want to expose it as filestream, or `event/filestream` via the mapper?

Reply:
Good point, I don't have a preference, `event/filestream` seems like a good idea? @mostlyjason @ruflin ?

Reply:
No strong preference on my end. Main downside of the prefix is that it is more cumbersome to type and harder to remember?

Reply:
No preference for me

Reply:
I would like us to maintain some kind of consistency for input names that are available via Agent. Unfortunately Filebeat input naming is not really consistent with the other Beats and even within Filebeat. As the input is not GA input I'm okay to keep it named `filestream`.

We already have `event/log` and the `logfile` inputs, each translating to `logs`. We will eventually replace `logs` with `filestream`. The `filestream` input is not 100% bc, that is we will also have to think about migration.

Reply:
Agree with the proposal @urso, can you give a review on this PR?